install/bin/decj.bat

```diff
@@ -21,4 +21,20 @@ for /R "%APP_PATH%/dep" %%a in (*.jar) do (
 
 set "CLASSPATH=!CLASSPATH!"
 
-java -Xmx240m -cp "!CLASSPATH!" -DDCSTOOL_HOME="%APP_PATH%" -DDECODES_INSTALL_DIR="%APP_PATH%" -DDCSTOOL_USERDIR="%APP_PATH%" %*%
+if not defined DCSTOOL_USERDIR (
+  set DCSTOOL_USERDIR="%APPDATA%\.opendcs"
+)
+
+if not exist %DCSTOOL_USERDIR%\ (
+  echo "Creating Local User Directory and initial properties in %DCSTOOL_USERDIR%"
+  echo "The default XML database has been copied to this directory."
+  mkdir %DCSTOOL_USERDIR%
+  copy %APP_PATH%\decodes.properties %DCSTOOL_USERDIR%\user.properties
+  copy /R %APP_PATH%\edit-db %DCSTOOL_USERDIR%\edit-db
```

Comment:
```suggestion
  xcopy %APP_PATH%\edit-db %DCSTOOL_USERDIR%\edit-db /E /I
```
unfortunately /R is not part of copy

xcopy /?

/E           Copies directories and subdirectories, including empty ones.
 /I           If destination does not exist and copying more than one file,
               assumes that destination must be a directory.